WcaOnRails/app/models/regulation.rb

```diff
@@ -0,0 +1,16 @@
+class Regulation < SimpleDelegator
+  begin
+    REGULATIONS = JSON.parse(File.read("app/views/search_results/wca-regulations.json"))
```

Comment:
I realized I did not totally handled @larspetrus comment: now that the overall integration of the feature is probably clearer with #747, any suggestion for the two points raised?